server/routes/landing.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const router = express.Router();
+const login = require('../controllers/login');
+const signup = require('../controllers/signup');
+const bcrypt = require('bcryptjs');
+
+router.post('/login', (req, res) => {
+  login.handleLogIn(req, res, bcrypt);
+});
```

Comment:
It would probably be cleaner to just import bcrypt in the files for the controllers rather than pass it as an argument

Reply:
Removed bcrypt from login and signup controller